src/Datadog.Trace/SpanTypes.cs

```diff
@@ -66,5 +66,10 @@ public static class SpanTypes
         /// The span type for msbuild integration.
         /// </summary>
         public const string Build = "build";
+
+        /// <summary>
+        /// The span type for azure function integration.
+        /// </summary>
+        public const string AzureFunction = "azure-func";
```

Comment:
These span types all look very generic. I wonder if there's any standard for these across tracers. e.g. is there a `serverless` span type? 🤔

Reply:
Not that I had seen but I will look again, I had tried a couple types/kinds and ended up getting the "custom" instrumentation indicators in the UI.
I'll check with some other tracers or the backend.

Reply:
There are only a few span types that have special meaning in the Agent or the backend/UI (I think `sql`, `redis`, `web`, and `http`). Anything else is considered a custom type.

Reply:
There is a serverless span type and in talking with the backend folks about lamda, that's the way to go.